packages/field/src/FormControlMixin.js

```diff
@@ -553,5 +571,96 @@ export const FormControlMixin = dedupeMixin(
       __getDirectSlotChild(slotName) {
         return [...this.children].find(el => el.slot === slotName);
       }
+
+      firstUpdated(changedProperties) {
+        super.firstUpdated(changedProperties);
+        this.__dispatchInitialModelValueChangedEvent();
+      }
+
+      __dispatchInitialModelValueChangedEvent() {
+        // When we are not a form-group / choice-group, we don't need to wait for our children
+        // to send a unified event
+        if (this._repropagationRole === 'child') {
+          return;
+        }
+
+        this.registrationComplete.then(() => {
```

Comment:
Nitpick but it would be clearer perhaps if this was its own async function called something like `async _repropagateAfterChildrenRegistered() {}`

Nice comment btw, helped me understand what this does 👍

Reply:
I already put it in a distinct function, before everything was in firstUpdated :)
Will change to `await this.registrationComplete;`